### PR TITLE
Added 2 second delay for sms/animations

### PR DESCRIPTION
**Pull request by kdsolo**
2 second delay added for hover/click animations and sending SMS

### CODE REVIEW

File: apps/system/js/haiku_emoji_panel.js [hunk 1]
```diff
@@ -8,6 +8,18 @@
   };
   var mozMessage = navigator.mozMobileMessage;
   var logStatus = document.querySelector('#emoji-status-bottom');
+  var handleNextSMS = true;
+  //sets handleSMS to true and enables hover animations
+  function setNextTrue() {
+    handleNextSMS = true;
+    document.getElementById('emoji_left').style.pointerEvents = 'auto';
+    document.getElementById('emoji_middle').style.pointerEvents = 'auto';
+    document.getElementById('emoji_right').style.pointerEvents = 'auto';
```
Comment (punamdahiya) [inline apps/system/js/haiku_emoji_panel.js]: we can add a class in css for pointer-events: none; and can remove that class from target elements here

File: apps/system/js/haiku_emoji_panel.js [hunk 2]
```diff
@@ -55,18 +67,27 @@
...
         return;
       }
 
       console.log('Haiku:sending emoji ', messageBody);
       mozMessage.send(receiver, messageBody);
+      }
+      //if it's < 2 seconds, disable hover/pointer (for animation) and do not send msg
+      document.getElementById('emoji_left').style.pointerEvents = 'none';
+      document.getElementById('emoji_middle').style.pointerEvents = 'none';
+      document.getElementById('emoji_right').style.pointerEvents = 'none';
```
Comment (punamdahiya) [inline apps/system/js/haiku_emoji_panel.js]: A css class e.g.  .avoid-clicks {
  pointer-events: none;
} can be used here. We can try adding it to parent section with id emoji-icons

Comment (kdsolo) [reply apps/system/js/haiku_emoji_panel.js]: ok!

File: apps/system/js/haiku_emoji_panel.js [hunk 3]
```diff
@@ -8,6 +8,16 @@
   };
   var mozMessage = navigator.mozMobileMessage;
   var logStatus = document.querySelector('#emoji-status-bottom');
+  var handleNextSMS = true;
```
Comment (sfoster) [inline apps/system/js/haiku_emoji_panel.js:11]: emojis.removeEventListener('click', this.emojiClickHandler);
setTimeout(() => {
  emojis.addEventListener('click', this.emojiClickHandler);
}, 2000);

**Comment by sfoster** [comment]
The adding/removing event listener wasnt working out (apologies.) I ended up implementing this using some state on the panel object: 51f565fe1b6b83c880b6fbaae23900c05ddb127f

**Pull request closed without merging**